src/e2e/app/cli/repo-tests/src/commands/test_ingest_command.rs

```diff
@@ -180,8 +184,289 @@ pub async fn test_push_ingest_from_file_snapshot_with_event_time(mut kamu: KamuC
+        &kamu,
+        [
+            "ingest",
+            "player-scores",
+            "--stdin",
+            "--source-name",
+            "default",
+        ],
+        DATASET_ROOT_PLAYER_SCORES_INGEST_DATA_NDJSON_CHUNK_2,
+        indoc::indoc!(
+            r#"
+            ┌────┬──────────────────────┬──────────────────────┬──────────┬───────────┬───────┐
+            │ op │     system_time      │      match_time      │ match_id │ player_id │ score │
+            ├────┼──────────────────────┼──────────────────────┼──────────┼───────────┼───────┤
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-01T00:00:00Z │        1 │     Alice │   100 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-01T00:00:00Z │        1 │       Bob │    80 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-02T00:00:00Z │        2 │     Alice │    70 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-02T00:00:00Z │        2 │   Charlie │    90 │
+            └────┴──────────────────────┴──────────────────────┴──────────┴───────────┴───────┘
+            "#
+        ),
+    )
+    .await;
+
+    assert_ingest_data_to_player_scores_from_stdio(
+        &kamu,
+        [
+            "ingest",
+            "player-scores",
+            "--stdin",
+            "--source-name",
+            "default",
+        ],
+        DATASET_ROOT_PLAYER_SCORES_INGEST_DATA_NDJSON_CHUNK_3,
+        indoc::indoc!(
+            r#"
+            ┌────┬──────────────────────┬──────────────────────┬──────────┬───────────┬───────┐
+            │ op │     system_time      │      match_time      │ match_id │ player_id │ score │
+            ├────┼──────────────────────┼──────────────────────┼──────────┼───────────┼───────┤
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-01T00:00:00Z │        1 │     Alice │   100 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-01T00:00:00Z │        1 │       Bob │    80 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-02T00:00:00Z │        2 │     Alice │    70 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-02T00:00:00Z │        2 │   Charlie │    90 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-03T00:00:00Z │        3 │       Bob │    60 │
+            │  0 │ 2050-01-02T03:04:05Z │ 2000-01-03T00:00:00Z │        3 │   Charlie │   110 │
+            └────┴──────────────────────┴──────────────────────┴──────────┴───────────┴───────┘
+            "#
+        ),
+    )
+    .await;
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Helpers
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
 fn path(p: &Path) -> &str {
     p.as_os_str().to_str().unwrap()
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+async fn assert_ingest_data_to_player_scores_from_stdio<I, S, T>(
```

Comment:
Optional for now: I think we can move it to a higher level. IT could be reused for different commands also

Reply:
Discussed in Slack: came to the conclusion that we don't need to